core/serialization/blocks.js

```diff
@@ -275,6 +296,7 @@ const load = function(state, workspace) {
  * @param {!Workspace} workspace The workspace to add the block to.
  * @param {!Connection} parentConnection The optional parent connection to
```

Comment:
I know it's not in this PR, but optional types need a `=` as in `!Connection=`

Reply:
Oh whoops! Thank you for the catch